bcipy/task/paradigm/rsvp/copy_phrase.py

```diff
@@ -744,6 +751,37 @@ def btn_press_count(self) -> int:
             activations = [inq for inq in inquiries if not inq.eeg_evidence]
         return len(activations)
 
+    def switch_response_time(self) -> Optional[float]:
+        """Computes the average switch response in seconds."""
+
+        # Remove consecutive items with the same type; we are only interested
+        # in PREVIEW followed by a EVENT.
+        triggers = destutter(self.switch_triggers(), key=lambda trg: trg.type)
+        pairs = list(zip(triggers[::2], triggers[1::2]))
+
+        # Confirm that the data is structured as expected.
+        for preview, keypress in pairs:
+            if (preview.type != TriggerType.PREVIEW) or (
+                    keypress.type != TriggerType.EVENT):
+                self.logger.info('Could not compute switch_response_time')
+                return None
+
+        response_times = [
+            keypress.time - preview.time for preview, keypress in pairs
+        ]
+        count = len(response_times)
+        return sum(response_times) / count if count > 0 else None
+
+    def switch_triggers(self) -> List[Trigger]:
+        """Returns a list of switch-related triggers"""
+        if not self.trigger_path:
+            return []
+        triggers, _offset = TriggerHandler.read_text_file(self.trigger_path)
+        return [
+            trg for trg in triggers
+            if trg.type in [TriggerType.PREVIEW, TriggerType.EVENT]
```

Comment:
This is currently working and gives us the appropriate triggers to use for the switch response annotation, but if we add any other triggers that use `TriggerType.EVENT` we need to remember to make the changes here to look for the symbol of interest.